pom.xml

```diff
@@ -3472,7 +3472,6 @@
               </targetClasses>
               <targetTests>
                 <param>com.puppycrawl.tools.checkstyle.grammar.*</param>
-                <param>com.puppycrawl.tools.checkstyle.JavaParserTest</param>
```

Comment:
`JavaParserTest` was only added here for the test that was moved to `JavaAstVisitorTest` in this PR.